### PR TITLE
Assignment 1 - ( Creating React Components , props , conditional rendering of components , useState hook ) 

### PR DESCRIPTION
**Pull request by rishabh1164**
Assignment 1 Questions

1.Create a functional component named Greeting that displays a simple "Hello, React!" message on the screen.
Import and render the Greeting component in the App component.

2.Create a functional component named UserCard that accepts the following props: name, email, and avatarURL.
Use the props to display the user's name, email, and an image (avatar) in the UserCard component.
Create multiple instances of the UserCard component with different user data.

3.Create a functional component named Weather that accepts a prop called temperature (a number).
Display a message like "It's sunny today!" if the temperature is above 25°C and "It's cold today!" if the temperature is below 10°C.
Import and render the Weather component in the App component with different temperature values.

4.Create a functional component named Counter that displays a count and two buttons: "Increment" and "Decrement".
Implement event handlers for the "Increment" and "Decrement" buttons to increase and decrease the count.
Display the updated count on the screen.

5.Create a functional component named TaskList that accepts an array of task names as a prop.
Use the map function to render each task name as a list item.
Import and render the TaskList component in the App component with an array of tasks.

6.Create a functional component named Button that accepts props for text and color.
Style the button using inline styles or CSS classes based on the color prop.
Import and render the Button component in the App component with different text and color props.

7.Create a functional component named CounterWithState that displays a count and buttons to increment and decrement it.
Use the useState hook to manage the count state.
Implement event handlers to update the count state accordingly.

### CODE REVIEW

File: assignment1/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "assignment1",
```
Comment (aviralnx) [inline assignment1/package.json:2]: Add a different name for eg. react-app

File: assignment1/src/App.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  const imageUrl3 =
+    "https://m.media-amazon.com/images/M/MV5BMTgwMTg4NDkyNF5BMl5BanBnXkFtZTcwMjc4MTUzNA@@._V1_UY100_CR39,0,100,100_AL_.jpg";
+  const temperature1 = 8;
+  const temperature2 = 25;
+  const temperature3 = 30;
+  const tasks = [
+    "React Assignment 1",
+    "Questions in JS Assignment 7",
+    "Coffee",
+  ];
```
Comment (aviralnx) [inline assignment1/src/App.js:25]: move the constants into a separate constant.js file

**Comment by aviralnx** [comment]
Provide a empty line in every file 
add semi colons as a best prfactice

**Comment by rishabh1164** [comment]
use prettier to format the files including adding a empty line and semi colons.

File: assignment1/src/App.js (hunk 2 above)
Comment (rishabh1164) [reply assignment1/src/App.js]: moved.

File: assignment1/package.json (hunk 1 above)
Comment (rishabh1164) [reply assignment1/package.json]: Name updated .

**Pull request merged**